awx/main/tasks/system.py

```diff
@@ -118,11 +122,7 @@ def inform_cluster_of_shutdown():
     try:
         this_inst = Instance.objects.get(hostname=settings.CLUSTER_HOST_ID)
         this_inst.mark_offline(update_last_seen=True, errors=_('Instance received normal shutdown signal'))
-        try:
-            reaper.reap_waiting(this_inst, grace_period=0)
-        except Exception:
-            logger.exception('failed to reap waiting jobs for {}'.format(this_inst.hostname))
```

Comment:
Wait, why'd you get rid of this? Won't waiting jobs get stuck in waiting forever if we don't reap them on shutdown? We could probably give a specific message - "job was in the queue while instance shut down", that would be good. But not reaping here would be bad.

Maybe you moved it somewhere I haven't found yet?

Reply:
This was a rebase conflict that was resolved inappropriately.  It should now be fixed.